### ISSUE TITLE
Microsoft.DotNet.Props was  not found.

### ISSUE THREAD

**Issue by jaredmoo** [Azure/azure-sdk-for-net#2922]
After installing VS2017, I always got build errors like this:

```
"D:\git\azure-sdk-for-net\src\ResourceManagement\SqlManagement\Sql.sln" (default target) (1) ->
"D:\git\azure-sdk-for-net\src\TestFramework\Microsoft.Azure.Test.HttpRecorder\HttpRecorder.xproj" (default target) (8)
->
  D:\git\azure-sdk-for-net\src\TestFramework\Microsoft.Azure.Test.HttpRecorder\HttpRecorder.xproj(7,3): error MSB4019:
The imported project "C:\Program Files\dotnet\sdk\1.0.1\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was
 not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
  D:\git\azure-sdk-for-net\src\TestFramework\Microsoft.Azure.Test.HttpRecorder\HttpRecorder.xproj(7,3): error MSB4019:
The imported project "C:\Program Files\dotnet\sdk\1.0.1\Microsoft\VisualStudio\v14.0\DotNet\Microsoft.DotNet.Props" was
 not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
```

The solution (after installing VS2015 build tools preview 2 as mentioned in [README](https://github.com/Azure/azure-sdk-for-net#prerequisites)) is to edit `globals.json` files to specify sdk version `1.0.0-preview2-003121`, e.g.

```
{
  "projects": [ "Microsoft.Rest.ClientRuntime", "Microsoft.Rest.ClientRuntime.Azure.Authentication", "Microsoft.Rest.ClientRuntime.Etw",
                "Microsoft.Rest.ClientRuntime.Tests", "Microsoft.Rest.ClientRuntime.Azure", "Microsoft.Rest.ClientRuntime.Azure.Tests",
                "Microsoft.Rest.ClientRuntime.Log4Net", "Microsoft.Rest.ClientRuntime.Tracing.Tests" ],
  "sdk": {
    "version": "1.0.0-preview2-003121"
  }
}
```

**Comment by jaredmoo** [comment]
See related:
https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/ProjectJsonToCSProj.md#how-do-i-work-with-projectjson-and-csproj-on-the-same-machine

This will be fixed by #2815, but it took me a while to find this solution.

**Comment by markcowl** [comment]
Closing as duplicate of #2815 this is scheduled for the current sprint.
